### ISSUE TITLE
Implement top down

### ISSUE THREAD

**Issue by maloufabmob** [fabmob/myc-emissions-calculator#2]
Adding the validation tool - second sheet of the excel

**Comment by maloufabmob** [comment]
- [ ] add first tab
- [ ] add second tab 
- [ ] add graph

**Comment by TTalex** [comment]
I'm wondering why is it divided between Road and Rail. Is fuel production different depending on the sector ?

I have mistakenly implemented a division between passengers and freight.  It will be pretty easy to unify it as one table if we decide that such division is not relevant.

Overall, I added topdown as a step before the overview. Users can however not fill anything and move on to the visualizations directly. According to IFEU, this step might be hard to fill for SUMPs, but can be. So displaying it makes sense. I have not added hard checks on the data, only visual ones. When there is a gap bigger than 20% between top-down and bottom-up approaches, a box becomes red, but users are free to move on. 

Some work also needs to be done on content, since I copied and pasted the long introduction from the Excel as a placeholder for now.
